README.md

```diff
@@ -1,7 +1,7 @@
 
-# api-platform-deskpro
+# API Platform Deskpro
 
-This is a placeholder README.md for a new repository
+Service for the creation of Deskpro Horizon tickets for use by API Platform
```

Comment:
Perhaps `Service for interfacing with Deskpro Horizon for use by API Platform`. The next step is to interface to organizations, so will someone remember to edit the README at that point?

Reply:
Done